src/Resources/app/storefront/src/configuration/adyen.js

```diff
@@ -22,7 +22,69 @@
 
 export default {
     updatablePaymentMethods: ['scheme', 'ideal', 'sepadirectdebit', 'oneclick', 'dotpay', 'bcmc'],
-    componentsWithPayButton: ['applepay', 'paywithgoogle', 'paypal'],
+    componentsWithPayButton: {
```

Comment:
moved some button config and event handlers here because paypal and google have quite different implementations